dwave_networkx/generators/chimera.py

```diff
@@ -449,6 +462,97 @@ def iter_linear_to_chimera_pairs(self, plist):
         """
         return self._pair_repack(self.iter_linear_to_chimera, plist)
 
+    def graph_to_linear(self, g):
+        """Return a copy of the graph g relabeled to have linear indices"""
+        labels = g.graph.get('labels')
+        if labels == 'int':
```

Comment:
Do we have two alternative nomenclatures with `linear` and `int` coordinates?

Reply:
Yes, I'm following the convention set in `chimera_graph`

Reply:
Ok, so `int` just gets used inside G.graph[`labels`] for Chimera, Pegasus, Zephyr. Outside of there we call them `linear`